packages/server/listenHandler.ts

```diff
@@ -4,8 +4,9 @@ import getGraphQLExecutor from './utils/getGraphQLExecutor'
 
 const listenHandler = (listenSocket: us_listen_socket) => {
   const PORT = Number(PROD ? process.env.PORT : process.env.SOCKET_PORT)
+  const SERVER_ID = Number(process.env.SERVER_ID)
```

Comment:
+1 Why do you parse it as number if you just want to print it?

Reply:
Did not test

Reply:
TBH, because I didn't know if it was needed and it was done that way the precedent line. I can change it if you want, I just tested it locally and has no impact.

Reply:
Please change it then. If for whatever reason this is not a number, it might be useful to see what it's set to. In your version it would simply print `NaN`.

Reply:
Actually, yesterday I had a similar problem with `const PORT = Number(PROD ? process.env.PORT : process.env.SOCKET_PORT)` 

I launched the application (in Kubernetes) without PORT env var configured and it just showed `NaN` in the message `Ready for Sockets: NaN` and then crashed. It might be worth to check somewhere if the mandatory parameters are set. Not in this PR, but what do you think @Dschoordsch?

Reply:
Yes, I'm all for it. Result will still be the application crashing, but at least with a better message :+1:

Reply:
Change done. I let you resolve it if it's OK